docs/UserGuide.md

```diff
@@ -17,72 +17,53 @@ Are you a developer? Make sure to check out our Developer Guide too! You can fin
       1. [Importing contacts `import`](#importing-contacts-import)
       2. [Exporting contacts `export`](#exporting-contacts-export)
-   5. [Advance](#aliasing-commands-alias)
-      1. [Aliasing commands `alias`](#aliasing-commands-alias)
-   6. [General](#exiting-the-program-exit)
-      1. [Exiting the program `exit`](#exiting-the-program-exit)
-      2. [Saving the data](#saving-the-data)
-      3. [Command History](#command-history)
+   5. [Aliasing commands `alias`](#aliasing-commands-alias)
+   6. [Exiting Socius `exit`](#exiting-socius-exit)
+   7. [Saving contacts](#saving-the-data)
+   8. [Accessing command history](#command-history)
 5. [Parameter Constraints](#parameter-constraints)
-   1. [`n/NAME`](#nname)
-   2. [`g/GENDER`](#ggender)
-   3. [`p/PHONE_NUMBER`](#pphonenumber)
-   4. [`e/EMAIL`](#eemail)
-   5. [`nat/NATIONALITY`](#natnationality)
-   6. [`tg/TUTORIAL_GROUP`](#tgtutorialgroup)
-   7. [`s/SOCIAL_HANDLE`](#ssocialhandle)
-   8. [`s/SOCIAL_HANDLE_USERID`](#ssocialhandleuserid)
-   9. [`r/REMARK`](#rremark)
-   10. [`t/TAG`](#ttag)
-   11. [`a/ALIAS`](#aalias)
-   12. [`c/COMMAND`](#ccommand)
 6. [FAQ](#faq)
 7. [Glossary](#glossary)
 8. [Authors](#authors)
 9. [Command Summary](#command-summary)
-   1. [General](#general)
-   2. [Modify](#modify)
-   3. [View](#view)
-   4. [Share](#share)
-   5. [Advance](#advance)
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Introduction to Socius
 
-Socius is a desktop application that can help you, as a CS2103T Software Engineering student, to
-* manage your classmates’ contacts
-* make friends and
+Socius is a desktop application that helps you, as a CS2103T Software Engineering student, to
+* manage your classmates’ contacts,
+* make friends, and
 * find teammates!
 
-The Socius User Guide helps you get started with using Socius.
-This User Guide hopes to get you familarized with the commands of Socius and the flow of using the program, helping you to achieve maximum efficiency so that you can get the most out of it.
-If you already have experience with using Socius, a summary table is available at the end for quick reference!
+The Socius User Guide gets you started with Socius.
+This User Guide familarises you with the commands and functionality of Socius, enabling you to get the most out of Socius.
+If you are an experienced Socius user, a command summary table is available at the end of the user guide for your reference!
 
 We hope that you will have a great time using Socius! :)
 
-Here are some main features Socius provides:
-* Access details of coursemates taking CS2103T.
-* Find any coursemate(s) with their name, tutorial group, nationalities, tags and more.
-* Tag coursemates with suitable tags to categorize them.
+Socius provides these main features:
+* Access details of students taking CS2103T.
+* Find any students with their name, tutorial group, nationality, tags, and more.
```

Comment:
Would "Find any student by..." sound better?

Reply:
Fixed. Thanks for vetting through the UG :)